### PR TITLE
Fix code block markup error and Markdown not rendering successfully on the issue

### PR DESCRIPTION
**Pull request by isolcat**
### Description

The page where the error occurred：https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/article

I have modified it by referring to the English document

### CODE REVIEW

File: files/zh-cn/web/html/element/article/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/HTML/Element/article
 
 {{HTMLSidebar}}
 
-**HTML `<article>`**元素表示文档、页面、应用或网站中的独立结构，其意在成为可独立分配的或可复用的结构，如在发布中，它可能是论坛帖子、杂志或新闻文章、博客、用户提交的评论、交互式组件，或者其他独立的内容项目。​​
+ **HTML `<article>`** 元素表示文档、页面、应用或网站中的独立结构，其意在成为可独立分配的或可复用的结构，如在发布中，它可能是论坛帖子、杂志或新闻文章、博客、用户提交的评论、交互式组件，或者其他独立的内容项目。​​
```
Comment (yin1999) [inline files/zh-cn/web/html/element/article/index.md:8]:
```suggestion
**HTML `<article>`** 元素表示文档、页面、应用或网站中的独立结构，其意在成为可独立分配的或可复用的结构，如在发布中，它可能是论坛帖子、杂志或新闻文章、博客、用户提交的评论、交互式组件，或者其他独立的内容项目。​​
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Comment by yin1999** [comment]
Welcome aboard 🎉

**Comment by isolcat** [comment]
> 欢迎登机🎉



> Welcome aboard 🎉

Really? I'd really like to join the MDN team.

**Comment by isolcat** [comment]
> Welcome aboard 🎉

How do I join your team?

**Comment by yin1999** [comment]
Hi @isolcat, thanks for the interest. But I'm afraid I could not propose for that.

Though that, we welcome developers from all over the world to participate in the MDN community.

This is a [getting started document](https://developer.mozilla.org/zh-CN/docs/MDN/Community/Contributing/Getting_started). And there is a [translation guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) for quick start.

**Comment by isolcat** [comment]
> Hi @isolcat, thanks for the interest. But I'm afraid I could not propose for that.
> 
> Though that, we welcome developers from all over the world to participate in the MDN community.
> 
> This is a [getting started document](https://developer.mozilla.org/zh-CN/docs/MDN/Community/Contributing/Getting_started). And there is a [translation guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) for quick start.

Well, I'm sorry I misunderstood your "Welcome aboard" earlier, I thought it was an invitation to join you, and I'm sorry about that.

**Comment by yin1999** [comment]
Just mean "Welcome to MDN community". Anyway, thanks for your help 🤗

**Pull request merged**